### PR TITLE
fix(compat): improve react version detection

### PR DESCRIPTION
**Pull request by acgrdumlu**
**What**:

**Why**:

**How**:

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [ ] Code complete
- [ ] Added myself to contributors table 
- [ ] Followed the commit message format

### CODE REVIEW

File: src/react-compat.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react'
 let PropTypes
 
 /* istanbul ignore next */
-if (React.version.slice(0, 4) === '15.5') {
+if (ParseFloat(React.version.slice(0, 4)) >= 15.5) {
```
Comment (kentcdodds) [inline src/react-compat.js]: Whoops! This should be a lower-case `p`. Sorry about that! So it should be:

```
if (parseFloat(React.version.slice(0, 4)) >= 15.5) {
```

**Comment by kentcdodds** [comment]
Oh, and another bit of feedback, normally when making a pull request, you'll want to fill in the Pull Request description with something useful. Some open source projects (like this one) will even have a template for you to fill out.

**Comment by acgrdumlu** [comment]
I cannot push again after the changes that you want me to. 
When I push the codes with this:
`git push --set-upstream origin pr/fix-react-version-detection-new` 
I'm giving this error;
remote: Permission to paypal/glamorous.git denied to acgrdumlu.
fatal: unable to access 'https://github.com/paypal/glamorous.git/': The requested URL returned error: 403

Could you help me about this issue?

**Comment by kentcdodds** [comment]
cc @vesparny (or anyone else), could you help @acgrdumlu? I'm afraid I don't have time (getting ready for family vacation!)

**Comment by ajwhite** [comment]
Make sure you push to your fork, @acgrdumlu. Sounds like you may be attempting to push to `paypal/glamorous`

**Comment by kwelch** [comment]
This is not resolved with the merge of #189, right?

**Comment by vesparny** [comment]
Yep. Closing

**Pull request closed without merging**